src/types/tablet_tool.zig

```diff
@@ -14,6 +14,18 @@ pub const TabletTool = extern struct {
         totem,
     };
 
+    pub const Axes = enum(c_int) {
+        x = 1 << 0,
+        y = 1 << 1,
+        distance = 1 << 2,
+        pressure = 1 << 3,
+        tilt_x = 1 << 4,
+        tilt_y = 1 << 5,
+        rotation = 1 << 6,
+        slider = 1 << 7,
+        wheel = 1 << 8,
+    };
+
```

Comment:
It probably makes more sense to make this a group of namespaced constants like e.g. `wlr.Output.State.field` until we update to zig 0.10.0 and can use a `packed struct(u32)`